### DISCUSSION TITLE
PaddleOCR is not returning any value on Jetson AGX Xavier

### DISCUSSION THREAD

**Discussion by MoyoG** [PaddlePaddle/PaddleOCR#9956] [General]
I use Jetson AGX Orin to develop a project.

Jetpack **5.0.1**
Python 3.8

Followed the instruction to install PaddlePaddle on Python 3.8, on **Jetpack 5.0.2**
https://forums.developer.nvidia.com/t/paddlepaddle-for-jetson/242765

The code doesn't throw any error, but the returning value is empty.

I would appreciate any leads or suggestions you might have.

**The same code works on my PC with GPU.**

**Comment by sminkwitz** [comment]
Hey MoyoG, i have the same problem.... have you sloved it?
